.env

```diff
@@ -2,6 +2,10 @@ POSTGRESQL_ADDRESS=localhost
 POSTGRESQL_USERNAME=metadataregistry
 POSTGRESQL_PASSWORD=metadataregistry
 POSTGRESQL_DATABASE=metadataregistry_development
+REDIS_URL=redis://localhost:6379/0
+GREMLIN_HTTP_ENDPOINT=https://localhost:8182/gremlin
+GREMLIN_USERNAME=credentialregistry
+GREMLIN_PASSWORD=8Zc7NJIMFDSrUIv
```

Comment:
Configuring credentials is not trivial for a Gremlin server, because they must be created inside a Kryo graph database. We're providing an example Gremlin configuration in the repo that includes the Kryo credentials database, meant for quick set up and tests. It won't be the same Kryo DB used in staging/production. The values in `.env` are for the Gremlin proxy endpoint in the API - I'll add more details about that in the documentation... anyway, it's not necessary to have those values in there, they're more for practical purposes. I can remove them.